src/Couchbase.Lite.Shared/API/Database/Database.cs

```diff
@@ -170,7 +174,8 @@ public sealed unsafe partial class Database : IDisposable
         public DatabaseConfiguration Config { get; }
 
         /// <summary>
-        /// Gets the number of documents in the database
+        /// [Obsolete("Count is deprecated, please use <see cref="GetDefaultCollection().Count"/>.")]
```

Comment:
Why is this commented?

Reply:
? Count in Database is marked deprecated in spec. And Obsolete comments will direct user to use the alternative way to get the Count. Did I miss anything?

Reply:
I mean, you have commented out the obsolete attribute, so it won't actually function like it should (giving compiler warning)

Reply:
woops... Will move them out of comments..